src/interactive-window/interactiveWindowProvider.ts

```diff
@@ -178,7 +181,7 @@ export class InteractiveWindowProvider
                 resource,
                 mode,
                 this.serviceContainer.get<IExportDialog>(IExportDialog),
-                this.notebookControllerManager,
+                this.serviceContainer.get<IControllerSelection>(IControllerSelection),
```

Comment:
Isn't this an indication of some cyclic reference (feels weird for the new code to have cyclic references)

Reply:
No it's not. I was just lazy. THere's already a bunch of serviceContainer calls, I didn't feel like importing another item into the provider.